oe_theme.theme

```diff
@@ -245,6 +245,18 @@ function oe_theme_preprocess_form_element(array &$variables): void {
  * Add paragraph formatting to body.
  */
 function oe_theme_preprocess_field(&$variables, $hook): void {
+  // Add space between paragraphs.
+  if (($variables['entity_type'] === 'node' &&
+      $variables['field_name'] === 'paragraphs' &&
+      $variables['element']['#bundle'] === 'landing_page') ||
```

Comment:
We can't simply hardcode bundle names in the theme, this is a generic component.

Reply:
I think this should not be tied to any bundle, spacing has to be correct on any bundle that has paragraph field.